### ISSUE TITLE
new v1 release?

### ISSUE THREAD

**Issue by sinistral** [23/resumable.js#328]
v1.0.2 was released ~2 years ago, and there have been several fixes and improvements since.  Do you have any plans to cut a new v1 release?

**Comment by thewilli** [comment]
would be great to especially publish a new [npm version](https://www.npmjs.com/package/resumablejs) as well

**Comment by steffentchr** [comment]
Yes, that should be done -- and I basically wanted to hold off with cutting the release until the most recent set of PRs had been merged. Any more coming, or should we call this a 1.1?

Thanks, 
Steffen

**Comment by hyperknot** [comment]
It's great to see that the newest commit is about bumping version number to 1.0.3! Is there anything why it's not a release on GitHub and NPM?

At the moment, I'm using

```
"resumablejs": "23/resumable.js#4a8ceb8"
```

As a workaround in package.json, but it'd be great to just use 1.0.3!

**Comment by steffentchr** [comment]
Someone else published the project to npm originally, and I'm waiting for ownership to be transferred in order for me to publish. So yeah, annoying to hard-reference a commit ;)

**Comment by hyperknot** [comment]
Wow, that's quite a reason! I wish you good luck with the transfer of the
ownership and thanks for the new release!

**Comment by k-schneider** [comment]
Is this still a thing? Why hasn't the npm package been updated to 1.0.3 yet?

**Comment by niklr** [comment]
I'm interested in this as well. What can we do to make it happen? ;)

**Comment by maximilianhuber** [comment]
I am currently trying to bump the version of resumable.js which is published via webjars at https://github.com/webjars/resumable.js. For this it would be very helpful if you could tag the commit https://github.com/23/resumable.js/commit/4a8ceb834f455629d73f9a00e24ab99bd7e48f81 as v1.0.3.

**Comment by steffentchr** [comment]
Absolutely -- done.

**Comment by maximilianhuber** [comment]
Thank you.

**Comment by tesujimath** [comment]
I'd also really like to see 1.0.3 available on npm.

6 months after your comment about the ownership transfer of the npm package, I wonder if there is still a blockage getting ownership transferred from @mspanc?  If so, would it be sensible to choose another package name, that could be the official resumablejs package on npm?

**Comment by cpnielsen** [comment]
@tesujimath (and others) We have been granted access to the repository, so you can expect a new release within the new few weeks.

Currently, we have to decide what to officially bump the version to (based on changes), and then get it packaged correctly and shipped off. I know this makes management of the package much easier, especially with the latest changes. Now that it is possible it's definitely something we are going to push.

**Comment by mspanc** [comment]
I passed ownership to @steffentchr and it seems mine is already revoked, so I am unable to publish 1.0.3

```
$ npm publish
npm ERR! publish Failed PUT 403
npm ERR! code E403
npm ERR! you do not have permission to publish "resumablejs". Are you logged in as the correct user? : resumablejs
```
